Description: Fix spelling errors identified by lintian.
 Lintian found some spelling errors in the binaries.
Author: Brian T. Smith <bsmith@systemfabricworks.com>
Last-Update: 2017-12-12
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/psm_ep_connect.c
+++ b/psm_ep_connect.c
@@ -132,7 +132,7 @@
 			if (psmi_epid_version(array_of_epid[j]) >
 						 PSMI_EPID_VERSION) {
 					psmi_handle_error(PSMI_EP_NORETURN, PSM2_INTERNAL_ERR,
-					  " Unkown version of EPID - %"PRIu64" \n"
+					  " Unknown version of EPID - %"PRIu64" \n"
 					  "Please upgrade PSM2 or set PSM2_ADDR_FMT=1 in the environment to force EPID version 1 \n",
 					  psmi_epid_version(array_of_epid[j]));
 			}
